Makefile.emscripten

```diff
@@ -125,7 +155,7 @@ libretro :=
 ifeq ($(HAVE_STATIC_DUMMY),1)
   DEFINES += -DHAVE_STATIC_DUMMY
 else
-  libretro += libretro_emscripten.bc
+  libretro += libretro_emscripten.a
```

Comment:
This needs to be changed in all repositories with emscripten targets: https://github.com/libretro/RetroArch/issues/16160#issuecomment-1913003712 if one core fails to build the entire ci will not be published

From what I've gathered the nightly ci does not use the dist-cores.sh #15545